### PR TITLE
Revisions

### PR DESCRIPTION
**Pull request by bossidy**
Substantial changes to the What RackHD Does Well Section

### CODE REVIEW

File: docs/introduction.rst [hunk 1]
```diff
@@ -115,28 +115,22 @@ capture sensor information and other data that can be retrieved using IPMI.
...
-data from the APIs that invoke workflows through to the configuration files that
-drive OS installs - like a kickstart or debseed file.
-
-While we have a number of workflows in our code repository as examples of how
-you can do a variety of actions, the real power of the system is in being able
-to create your own workflows - and submit those through the REST API. So you
-can define arbitrary workflows for your needs, specific to your hardware if
-needed, to accomplish your automation goals.
+RackHD is focused on being the lowest level of automation that interrogates agnostic hardware,
+provisions machines with operating systems. With SKU support, the workflow engine can react to
```
Comment (heckj) [inline docs/introduction.rst]: I think the first sentence reads a touch cleaner with something like:

"RackHD is focused on being the lowest level of automation that interrogates agnostic hardware and 
provisions machines with operating systems."

File: docs/introduction.rst [hunk 2]
```diff
@@ -115,28 +115,22 @@ capture sensor information and other data that can be retrieved using IPMI.
...
+information found during discovery and respond with dynamic rendering of installation templates.
+During this process, the API can be used to pass variables and data to the configuration files
```
Comment (heckj) [inline docs/introduction.rst]: Drop the "during this process" - as it implies (to me) that when the workflow is running you can interact with it via an API call to pass in additional information, which you can't. 

Maybe instead "The API can be used to pass in data through variables in the workflow configuration, so you can parameterize your workflows. The workflows also have access to all of the SKU information and other catalogs, so workflows can be authored that react to that information"

Comment (bossidy) [inline docs/introduction.rst]: OK

File: docs/introduction.rst [hunk 3]
```diff
@@ -115,28 +115,22 @@ capture sensor information and other data that can be retrieved using IPMI.
...
+that drive OS installs. This method can be used, for example, to include a kickstart or debseed file.
+
```
Comment (heckj) [inline docs/introduction.rst]: dropped the poller piece, so maybe something like 

"RackHD is also good at capturing data from hardware periodically and publishing that information as a "live data stream of events" for higher levels of orchestration to use."

**Comment by bossidy** [comment]
Finishing up for the day, but will give it another go over tomorrow and also look at the What RackHD Doesn't Do" section...

**Comment by bossidy** [comment]
After I did a merge, I used the same branch to do a few more changes. After I did the second Commit, Git Desktop doesn't give me a chance to do the Pull that sends it to to GitHub. 

I will look at this tomorrow to correct any problems.

**Pull request merged**